lib/katello/tasks/upgrades/3.0/update_subscription_facet_backend_data.rake

```diff
@@ -13,11 +36,22 @@ namespace :katello do
             subscription_facet.host = ::Host::Managed.find(subscription_facet.host_id)
             subscription_facet.save!
 
+            if subscription_facet.host.name.include?('_')
+              host = subscription_facet.host
+              host.name.gsub!('_', '-')
```

Comment:
might want to throw in a lowercase here too, had some situations where the name had upper case chars